src/tracer.js

```diff
@@ -76,22 +77,23 @@ export const startTrace = async () => {
   }
 };
 
+const logLeakedSpans = allSpans => {
+  const warnClientOnce = runOneTimeWrapper(logger.warnClient);
+  allSpans.forEach(span => {
+    if (isSpanIsFromAnotherInvocation(span)) {
+      logger.debug('Leaked span: ', span);
+      warnClientOnce(LEAK_MESSAGE);
```

Comment:
This is in `forEach` but it is wrapped by `runOneTimeWrapper`

Reply:
Yes, we want to warn the client just once per invocation, it's instead of the counter-loop that was before